.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto
+*.js text eol=lf
```

Comment:
any reason not to us `eol=auto` ?

Reply:
related to: https://github.com/ovh-ux/manager-webpack-config/blob/master/.editorconfig#L15